### ISSUE TITLE
Unknown field "bufferline" for key "editor"

### ISSUE THREAD

**Issue by IowU** [helix-editor/helix#5039]
Hello,
on the [documentation](https://docs.helix-editor.com/configuration.html#editor-section) page for the helix `config.toml` file, in the editor section it is reported that there's a key named `bufferline` that can be set.

If I then try to set it up, write and quit the changes, exit helix and then starting it again, I get the following error message

```
Bad config: unknown field `bufferline`, expected one of `scrolloff`, `scroll-lines`, `mouse`, `shell`, `line-number`, `cursorline`, `gutters`, `middle-click-paste`, `auto-pairs`, `auto-completion`, `auto-format`, `idle-timeout`, `completion-trigger-len`, `auto-info`, `file-picker`, `statusline`, `cursor-shape`, `true-color`, `search`, `lsp`, `terminal`, `rulers`, `whitespace`, `indent-guides`, `color-modes` for key `editor` at line 26 column 1
Press <ENTER> to continue with default config
```

From the keys displayed by the error, the `bufferline` is not showing up, while the docs say it should.

My editor section in the `config.toml` file looks like this

```
theme = "monokai_pro"

[editor]
line-number = "relative"
bufferline = "always"
```

I reproduced it on both a Windows Machine, and a Linux one (using the fedora distro).

**Comment by archseer** [comment]
`hx -v`?

**Comment by IowU** [comment]
That gives me the same message on screen. I checked the log file, but nothing is being written

**Comment by archseer** [comment]
What helix version are you running? Bufferline was added in a release just hours ago so I'm assuming you're on 22.08 still. hx -v should print out the version

**Comment by IowU** [comment]
Running `hx -v` doesn't print anything; I checked the logs again, and the version isn't shown.
I tried the comand `hx --version`, and this is the output:

```
hx --version
helix 22.08.1 (66276ce6)
```

**Comment by archseer** [comment]
Ah sorry, I gave you the wrong flag. Yeah you're on the version released in August and you need to update to 22.12 (released today).

**Comment by IowU** [comment]
Thanks. I'll wait until the new version is available on my package manager and will try to update it.
